js/components/main.js

```diff
@@ -125,12 +126,15 @@ class Main extends ImmutableComponent {
         yVelocity = deltaY / time
       }
     })
+    ipc.on(messages.GOT_SCROLL, (e) => {
+      isScroll = true
+    })
     ipc.on('scroll-touch-begin', function () {
       trackingFingers = true
       startTime = (new Date()).getTime()
     })
     ipc.on('scroll-touch-end', function () {
-      if (trackingFingers && Math.abs(yVelocity) < 1) {
+      if (!isScroll && trackingFingers && Math.abs(yVelocity) < 1) {
         if (xVelocity > 1.5) {
```

Comment:
I think you can change the xVelocity threshold to 1 or 0.5 to make it easier to go back/forward now